es-vue-base/src/lib-components/EsFileInput.vue

```diff
@@ -201,6 +201,10 @@ export default {
             this.pickedItems = [];
         },
         async verifyMimeType(file) {
+            // If a folder then trigger fileTypeError
+            if (file.type === '') {
+                this.$emit('fileTypeError', file.name);
```

Comment:
You could make this something like `fileIsAFolderError` or maybe emit a boolean flag to indicate if the error is due to the file being read as a folder?

Reply:
thank you for your comments, made the changes